webui/react/src/shared/hooks/useModal/useModal.tsx

```diff
@@ -49,16 +49,6 @@ interface ModalOptions {
  * If you skip the clone step, when attempting to re-open the
  * same modal to show the "cannot delete" message will fail.
  */
```

Comment:
can this comment be removed with `CANNOT_DELETE_MODAL_PROPS`?

Reply:
✅ , also `DEFAULT_MODAL_PROPS` is used only in this file so I will move those props